src/gvizdoom_client/App.cpp

```diff
@@ -96,8 +96,8 @@ void App::loop(void)
         }
 #endif
         // Update game
-        //_quit = doomGame.update(actions.at(actionIndex++));
-        _quit = _quit || doomGame.update(_actionMapper);
+        _quit = doomGame.update(actionIndex < actions.size() ? actions.at(actionIndex++) : Action());
+        // _quit = _quit || doomGame.update(_actionMapper);
```

Comment:
So now the client is hard-coded for this test case? Maybe utilize the `interactive` flag in `GameConfig` to select between the two?